types/react-slider/index.d.ts

```diff
@@ -25,6 +25,15 @@ export interface ReactSliderProps<T extends number | ReadonlyArray<number> = num
      */
     ariaLabel?: T extends number ? string : ReadonlyArray<string>;
 
+
+    /**
+     * aria-labelledby for screen-readers to apply to the thumbs.
+     * Used when slider rendered with separate label.
+     * Use an array for more than one thumb.
+     * The length of the array must match the number of thumbs in the value array.
+     */
+    ariaLabelledby?: string | string[];
```

Comment:
It should probably be `T extends number ? string : ReadonlyArray<string>` like for `ariaLabel`

Reply:
fixed